### ISSUE TITLE
commit info not automatically shown in some cases (git 2.4.3)

### ISSUE THREAD

**Issue by linquize** [FredrikNoren/ungit#580]
I compile ungit source code and run it
```
user@linux:~$
git clone git://github.com/FredrikNoren/ungit.git
cd ungit
npm install
grunt
bin/ungit
```

ungit is browsing ungit repository
The dotted circle appears.
commit info of past commits automatically shown. No need to click the commit node.

But if I browse another repository, say `/home/user/abc`
The dotted circle disappears.
commit info of past commits not automatically shown. Need to click the commit node.

**Comment by campersau** [comment]
The commit info is automatically shown if the commit is on your current branch.
Maybe your newest commits are on different branches so the info of these commits is hidden.

Here is a screenshot which demonstrates it:
![image](https://cloud.githubusercontent.com/assets/4009570/8135279/5ca6c338-1136-11e5-9465-61b55d745161.png)

The dotted circle disapears if you have a lot of newer commits on other branches. Just scroll down until you see the current branch.

If this is not the case could you please post a screenshot which shows the problem?

**Comment by linquize** [comment]
A simple repository (`/home/user/abc`) with the only branch master does not show like above screenshot.
Only ungit repository works if viewing `/home/user/ungit` when running my compiled `/home/user/ungit/bin/ungit`

**Comment by FredrikNoren** [comment]
Fixed in #581
